app/gcl/agwa/section2/CollapsibleSection.tsx

```
@@ -0,0 +1,65 @@
+    const handleKey = (event: KeyboardEvent<HTMLDivElement>) => {
+        if (event.key === 'Enter' || event.key === ' ') {
+            event.preventDefault();
+            toggle();
+        }
+    };
+
+    return (
+        <div className={styles['section-card']} id={id} style={wrapperStyle}>
+            <div
+                className={styles['section-header']}
+                role="button"
```

Biome (2.4.15)

[error] The elements with this role can be changed to the following elements:
<button>

(lint/a11y/useSemanticElements)